alembic/versions/1ab5aa1e3054_start_associating_comments_with_.py

```diff
@@ -0,0 +1,28 @@
+"""Start associating comments with registered users and use user information
```

Comment:
I'd frame these migration notes as 1) describing the change on the DB (e.g. "Add a user_id column to user_comments and drop author_name.") 2) then describe why (e.g. "This allows us to associate comments with users in the database.")